### PR TITLE
update wsSecurity feature and runtime bundles to work with jakarta ee9

### PR DESCRIPTION
**Pull request by arunavemulapalli**
Start with wsSecurity updates.

### CODE REVIEW

**Comment by arunavemulapalli** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat.wsscxf

File: dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.wsSecurity1.1.javaee.feature [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName = io.openliberty.wsSecurity1.1.javaee
+visibility = private
+IBM-Provision-Capability:\
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.wsSecurity-1.1))",\
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.servlet.api-3.0)(osgi.identity=io.openliberty.servlet.api-3.1)(osgi.identity=io.openliberty.servlet.api-4.0)))"
+IBM-Install-Policy: when-satisfied
+-features=com.ibm.websphere.appserver.jta-1.1; ibm.tolerates:="1.2", \
+  io.openliberty.servlet.api-3.0; apiJar=false; ibm.tolerates:="3.1,4.0"
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.wsSecurity1.1.javaee.feature]: I don't think you need servlet.api-3.0, ... listed as a dependent feature.

File: dev/com.ibm.ws.org.apache.wss4j.ws.security.dom.2.3.0/src/org/apache/wss4j/dom/util/WSSecurityUtil.java [hunk 2]
```diff
@@ -54,6 +54,8 @@
 
 import javax.security.auth.callback.CallbackHandler;
 
+import com.ibm.ws.ffdc.annotation.FFDCIgnore; //Liberty code change
```
Comment (jhanders34) [inline dev/com.ibm.ws.org.apache.wss4j.ws.security.dom.2.3.0/src/org/apache/wss4j/dom/util/WSSecurityUtil.java:57]: Or should we not ffdc instrument the classes in the CXF code?

**Review by jhanders34 — APPROVED** [review]

File: dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.wsSecurity1.1.javaee.feature [hunk 3]
```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName = io.openliberty.wsSecurity1.1.javaee
+visibility = private
+IBM-Provision-Capability:\
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.wsSecurity-1.1))",\
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.servlet.api-3.0)(osgi.identity=io.openliberty.servlet.api-3.1)(osgi.identity=io.openliberty.servlet.api-4.0)))"
+IBM-Install-Policy: when-satisfied
+-features=com.ibm.websphere.appserver.jta-1.1; ibm.tolerates:="1.2", \
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.wsSecurity1.1.javaee.feature]: Should this have apiJar=false like it did before?

File: dev/com.ibm.ws.org.apache.wss4j.ws.security.dom.2.3.0/src/org/apache/wss4j/dom/util/WSSecurityUtil.java (hunk 2 above)
Comment (arunavemulapalli) [reply dev/com.ibm.ws.org.apache.wss4j.ws.security.dom.2.3.0/src/org/apache/wss4j/dom/util/WSSecurityUtil.java:57]: I see that in the existing cxf ws-security project’s bnd.bnd files, we have this setting instrument.ffdc set to “true” and then have a set of classes specified in
 the instrument.classesIncludes .. So, I copied these as is to the new project’s bnd files.
I will need to investigate this a bit more (my guess is that the some of the fat tests may fail if they expect the FFDC) and then decide whether or not to ffdc instrument these cxf classes.

File: dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.wsSecurity1.1.javaee.feature (hunk 1 above)
Comment (arunavemulapalli) [reply dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.wsSecurity1.1.javaee.feature]: okay, removed it.

File: dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.wsSecurity1.1.javaee.feature (hunk 3 above)
Comment (arunavemulapalli) [reply dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.wsSecurity1.1.javaee.feature]: done.

**Comment by arunavemulapalli** [comment]
#build

**Comment by arunavemulapalli** [comment]
#run-libby-bot

**Comment by arunavemulapalli** [comment]
for #12449

**Comment by arunavemulapalli** [comment]
issue #16460

**Pull request merged**